xen.spec.in

```diff
@@ -964,6 +943,7 @@ rm -rf %{buildroot}
 %defattr(-,root,root)
 %ifnarch armv7hl aarch64
 /boot/xen-*.gz
+/boot/xen.gz
```

Comment:
The bare `xen.gz` was removed in our package quite intentionally. Otherwise there would be multiple entries in grub (with and without version).